### ISSUE TITLE
/status table with no USD amounts and no USD total amount

### ISSUE THREAD

**Issue by myNextCoder** [freqtrade/freqtrade#6518]
## Describe your environment

  * Operating system: Windows Server 2022
  * Python Version: _____ (`python -V`)
  * CCXT version: _____ (`pip freeze | grep ccxt`)
  * Freqtrade Version: 2022.2.1
  
## Your question

It seems already to be public, that some users with '/status table' do not see any USD amounts and no USD total amount. On my side this issue took place for the first time on March, 9th, with version 2022.1. And today, March 10th, with version 2022.2.1.
In the log I see the known message ". . .multiple mappings in goingekko for usdt. . .unsupported crypto-symbol usdt. . ."
If you already have a workaround, let me know. I thank you in advance.

*Ask the question you have not been able to find an answer in the [Documentation](https://www.freqtrade.io/en/latest/)*

**Comment by xmatthias** [comment]
This has been fixed in 6e10439f908a11a41501b3dad216f1ad8e90b80a - so will be available if you're using develop.
The problem happens because coingecko (or the crypto community in general?) is running out of "coin symbol names" - so there's another "usdt" now - namely `Tether Avalanche Bridged (USDT.e)`.

We can explicitly map common ones for now (which the above commit does) - but it's tedious as it can happen at any time - with any coin.

**Comment by myNextCoder** [comment]
I thank you, this sounds great, so I use the develop version from now on instead of the stable one.

**Comment by myNextCoder** [comment]
I have updated to develop version 24f480b, which contains the updated fiat_convert.py. Unfortunately with dryrunning on gate.io the bot does not start, saying, that any currency pair I place, is not present on the exchange.
What I did: I used the stable version 2022.2.1, by only replacing the file fiat_convert.py. The bot starts again, and is also showing me all USD amounts, as I fire /status table.
